elasticsearch/utils/logs

```diff
@@ -0,0 +1,57 @@
+#!/bin/bash -e
+#
+# Copyright 2017 Red Hat, Inc. and/or its affiliates
+# and other contributors as indicated by the @author tags.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+set -euo pipefail
+log_dir=/elasticsearch/logging-es/logs
```

Comment:
this should actually change based on whether or not it is an ops cluster. Instead of hardcoding `logging-es-` here and in `file` can we pull from the ENV to know the cluster name instead?

Reply:
Why aren't we just using `/elasticsearch/logs`?

Reply:
The initial thought was in case a user wants to use the same storage for ops and non-ops... so its been `/elasticsearch/$CLUSTER_NAME/logs` since... its configured in the configmap that way presently.